internal/errors.go

```diff
@@ -4,6 +4,7 @@ const (
 	ErrAlreadyUpToDate     = "Already up to date"
 	ErrLoginFailed         = "Login failed"
 	ErrNoActiveDevice      = "Player command failed: No active device found"
+	ErrNoAlbums            = "No albums found"
 	ErrNoDevices           = "No devices"
 	ErrNoPlaylists         = "No playlists"
```

Comment:
Let's rename these for consistency.

```suggestion
	ErrNoAlbumsFound            = "No albums found"
	ErrNoDevicesFound           = "No devices found"
	ErrNoPlaylistsFound         = "No playlists found"
```